app/controllers/downvotes_controller.rb

```diff
@@ -0,0 +1,19 @@
+class DownvotesController < ApplicationController
+  before_action :require_login, only: [:create]
+
+  def create
+    @vote = parent.votes.for_user(current_user).first_or_initialize
+    @vote.score = -1
+    @vote.save
```

Comment:
Worth noting that `#update_attribute` skips validations. A one-liner that does run validations would be `@vote.update_attributes(score: -1)`